syntax="proto2";

package peleus.plugins;

message HttpRequest {
    required string conn_name = 1;
    required string uri = 2;
    required bool post = 3;
    optional string post_data = 4;
};

message HttpResponse {
    required int32 error_code = 1;
    required string error_text = 2;
    optional string response = 3;
};

